auth/middleware.go

```diff
@@ -59,3 +72,25 @@ func (k *kratosMiddleware) validateSession(r *http.Request) (*client.Session, er
 	}
 	return session, nil
 }
+
+func (k *kratosMiddleware) generateDBToken(id string) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
+		"role": DefaultPostgrestRole,
```

Comment:
I think we need an expiry and then use that same expiry in the cache:
```
		// A usual scenario is to set the expiration time relative to the current time
		ExpiresAt: jwt.NewNumericDate(time.Now().Add(24 * time.Hour)),
```

Reply:
@moshloop Since the cache key works on sessionID, and users never have access to the token, we don't have to worry about expiring the token.

Postgrest in itself supports expiry by using `exp: unix epoch time`, but if we do that, we have to check everytime if a token is expired or not.

The current approach just uses sessionID which is always validated first, and when the session expires a new token is created for that session+user  and the cache's default expiry will cleanup the old token